scripts/db-heal-test.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
```

Comment:
a documentation how to use the script should be added at the beginning of the script. For example, it is not clear where the script should be run (in Aida, in Carmen, ...) and what the input parameters are.